library/alloc/src/collections/btree/node/tests.rs

```diff
@@ -94,10 +94,10 @@ fn test_partial_eq() {
 
 #[test]
 #[cfg(target_arch = "x86_64")]
+#[cfg(not(feature = "randomized-layouts"))]
 fn test_sizes() {
-    // FIXME: these make assumptions about repr(Rust) layout which doesn't hold under -Zrandomize-layout
-    // assert_eq!(core::mem::size_of::<LeafNode<(), ()>>(), 16);
-    // assert_eq!(core::mem::size_of::<LeafNode<i64, i64>>(), 16 + CAPACITY * 2 * 8);
-    // assert_eq!(core::mem::size_of::<InternalNode<(), ()>>(), 16 + (CAPACITY + 1) * 8);
-    // assert_eq!(core::mem::size_of::<InternalNode<i64, i64>>(), 16 + (CAPACITY * 3 + 1) * 8);
+    assert_eq!(core::mem::size_of::<LeafNode<(), ()>>(), 16);
+    assert_eq!(core::mem::size_of::<LeafNode<i64, i64>>(), 16 + CAPACITY * 2 * 8);
+    assert_eq!(core::mem::size_of::<InternalNode<(), ()>>(), 16 + (CAPACITY + 1) * 8);
+    assert_eq!(core::mem::size_of::<InternalNode<i64, i64>>(), 16 + (CAPACITY * 3 + 1) * 8);
```

Comment:
nit: it would be nice to squash this into the previous commit, was reviewing commit-by-commit and got confused for a bit